routes/badges.js

```diff
@@ -302,7 +299,7 @@ module.exports = function (env) {
         if (err) {
           var errorString = err.message;
           if (err.code === 404) {
-            errorString = 'The code "' + req.body.claimcode + '" could not be found';
+            errorString = 'The code \'' + req.body.claimcode + '\' could not be found';
```

Comment:
this substitution should not be necessary, it's a wrapped double quote. If JSCS, JSHint or JSBeautify tripped up on this, we need to investigate that.